source/common/upstream/health_discovery_service.h

```diff
@@ -41,25 +112,49 @@ class HdsDelegate
       std::unique_ptr<envoy::service::discovery::v2::HealthCheckSpecifier>&& message) override;
   void onReceiveTrailingMetadata(Http::HeaderMapPtr&& metadata) override;
   void onRemoteClose(Grpc::Status::GrpcStatus status, const std::string& message) override;
-
+  envoy::service::discovery::v2::HealthCheckRequestOrEndpointHealthResponse sendResponse();
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_DELAY_MS = 5000;
+  const uint32_t RetryDelayMilliseconds = 5000;
+  static constexpr uint32_t ClusterConnectionBufferLimitBytes = 12345;
+  static constexpr uint32_t ClusterTimeoutSeconds = 1;
```

Comment:
Can you add comments for all of these constants to explain what they do/represent?